### ISSUE TITLE
Observable Condition implementation

### ISSUE THREAD

**Issue by btilbrook-nextfaze** [NextFaze/power-adapters#23]
Create a `Condition` with a value derived from a `Observable<Boolean>`.

This should live in a new module `power-adapters-rx`.

**Comment by cheneveld** [comment]
I am having trouble getting at RXCondition.  I am including the following in gradle:

`compile 'com.nextfaze.poweradapters:power-adapters:0.12.1'
    compile 'com.nextfaze.poweradapters:power-adapters-recyclerview-v7:0.12.1'
    compile 'com.nextfaze.poweradapters:power-adapters-data:0.12.1'
    compile 'com.nextfaze.poweradapters:power-adapters-data-rx:0.12.1'`

And tried `compile 'com.nextfaze.poweradapters:power-adapters-rx:0.12.1'` with no luck.

Any help appreciated. thanks

**Comment by btilbrook-nextfaze** [comment]
Hey. `RxCondition` isn't implemented in `0.12.1`. However, you can access it by grabbing the latest Power Adapters via JitPack. Here's how:

```
repositories {
    ...
    maven { url 'https://jitpack.io' }
    ...
}
```

```
compile "com.github.NextFaze.power-adapters:power-adapters:a34e735"
compile "com.github.NextFaze.power-adapters:power-adapters-rx:a34e735"
compile "com.github.NextFaze.power-adapters:power-adapters-data-rx:a34e735"
compile "com.github.NextFaze.power-adapters:power-adapters-recyclerview-v7:a34e735"
```

This version also has some Kotlin modules.

Here's more information on how to use JitPack: https://jitpack.io/#NextFaze/power-adapters

I've found it to be a bit flaky sometimes. Try syncing a few times if you have trouble.

I intend on releasing a new version (`0.13.0`) to Maven Central shortly. Hopefully in the next few days if I've got time!

**Comment by cheneveld** [comment]
Thanks
